<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>465</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>decaf-qt</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/images/icon</normaloff>:/images/icon</iconset>
  </property>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>675</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionSystemSettings"/>
    <addaction name="actionInputSettings"/>
    <addaction name="actionLogSettings"/>
    <addaction name="actionDebugSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
     <addaction name="actionViewFullScreen"/>
     <addaction name="separator"/>
     <addaction name="actionViewSplit"/>
    <addaction name="actionViewTV"/>
    <addaction name="actionViewGamepad1"/>
    <addaction name="actionViewGamepad2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionInputSettings">
   <property name="text">
    <string>&amp;Input Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionSystemSettings">
   <property name="text">
    <string>S&amp;ystem Settings</string>
   </property>
  </action>
  <action name="actionDebugSettings">
   <property name="text">
    <string>&amp;Debug Settings</string>
   </property>
  </action>
  <action name="actionLogSettings">
   <property name="text">
    <string>&amp;Log Settings</string>
   </property>
  </action>
  <actiongroup name="viewActionGroup">
    <action name="actionViewFullScreen">
      <property name="text">
        <string>Full Screen</string>
      </property>
    </action>
    <action name="actionViewSplit">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Split</string>
    </property>
   </action>
   <action name="actionViewTV">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>TV</string>
    </property>
   </action>
   <action name="actionViewGamepad1">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Gamepad 1</string>
    </property>
   </action>
   <action name="actionViewGamepad2">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Gamepad 2</string>
    </property>
   </action>
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </actiongroup>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionInputSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openInputSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openAboutDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionViewGamepad1</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>setViewModeGamepad1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
   <connection>
     <sender>actionViewFullScreen</sender>
     <signal>triggered()</signal>
     <receiver>MainWindow</receiver>
     <slot>toggleFullscreen()</slot>
     <hints>
       <hint type="sourcelabel">
         <x>-1</x>
         <y>-1</y>
       </hint>
       <hint type="destinationlabel">
         <x>337</x>
         <y>232</y>
       </hint>
     </hints>
   </connection>
   <connection>
   <sender>actionViewSplit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>setViewModeSplit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionViewTV</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>setViewModeTV()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLogSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openLoggingSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDebugSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openDebugSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSystemSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openSystemSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openInputSettings()</slot>
  <slot>openFile()</slot>
  <slot>openAboutDialog()</slot>
  <slot>exit()</slot>
  <slot>setViewModeSplit()</slot>
  <slot>setViewModeTV()</slot>
  <slot>setViewModeGamepad1()</slot>
  <slot>openLoggingSettings()</slot>
  <slot>openSettings()</slot>
  <slot>openDebugSettings()</slot>
  <slot>openSystemSettings()</slot>
 </slots>
</ui>
